compact/pid/dirc.xml

```diff
@@ -56,7 +57,6 @@
     <!-- Bar - Each DIRC box consists of N "bars" -->
     <!-- BarAssembly - Bars + Glue -->
     <constant name="DIRCBarAssm_length"     value="DIRC_length - DIRCPrism_length - DIRCMirror_thickness - DIRCLens_thickness - DIRCMCP_thickness" comment="Length of bars+glue assembly"/>
-    <constant name="DIRCBar_count_y"        value="10" comment="Number of bars per box in y direction"/>
```

Comment:
"DIRCBarAssm_length" uses "DIRCLens_thickness" so moving the lens section after the bar section would not work here. We may just use "DIRCLens_width = DIRCPrism_width / 10" instead of "DIRCBar_count_y".

Reply:
No, you should use `DIRCBar_count_y` instead of a hard-coded number 10. We don't want magical constants floating around. There are no circular dependencies here (there can't be), so it's just a matter of figuring out the ordering.